designs/UI/business_creation.html

```diff
@@ -0,0 +1,110 @@
+    <script src="main.js"></script>
+    <!-- for font awesome -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+        <nav class="navbar navbar-expand-lg navbar-light bg-light navbar-dark bg-dark" style="height:50px">
+                <div class="row"></div>
+        </nav>
+        
+    <div class="container-fluid">
+        <div class="row">
+            <div class="col-xs-6 col-sm-2"></div>
+            <div class="col-xs-6 col-sm-8">
+                <div >
+                 <h3 class="text-center text-primary font-weight-bold">Business Registration Form</h3>
+             </div>
+                <hr>
+            </div>
+            <div class="col-xs-6 col-sm-2"></div>
+        </div>
+    </div>
+    <div class="container-fluid">
+        <div class="row">
+            <div class="col-xs-6 col-sm-2"></div>
+            <div class="col-xs-6 col-sm-8">
+            <div class="card border-default mb-4" style="">
+                <div class="card-header">Business Registration</div>
+                <div class="card-body">
+                    <form class="">
+                    
+                        <div class="form-group">
+                                <div class="text-left" style="">Business Name</div>
+                                <div class="input-group">
+                                    <div class="input-group-prepend">
+                                    </div>
+                                    <input type="text" class="form-control" placeholder="e.g latifajoy" aria-label="samsonanami" aria-describedby="basic-addon1">
+                                    </div>
+                        </div>
+
+                        <div class="form-group">
+                                <div class="text-left" style="">Location</div>
+                                <div class="input-group">
+                                    <div class="input-group-prepend">
+                                    </div>
+                                    <input type="text" class="form-control" placeholder="e.g Samwel Joyk" aria-label="samsonanami" aria-describedby="basic-addon1">
+                                  </div>
+                        </div>
+
+                        <div class="form-group">
+                                <div class="text-left" style="">Email Address</div>
+                                <div class="input-group">
+                                    <div class="input-group-prepend">
+                                    </div>
+                                    <input type="text" class="form-control" placeholder="e.g samwelemail@gmail.com" aria-label="samsonanami" aria-describedby="basic-addon1">
+                                  </div>
+                        </div>
+
+                        <div class="form-group">
+                                <div class="text-left" style="">Phone number</div>
```

Comment:
Since you only displaying text, it's highly advised you use `<p>` or `<span>` tag

Reply:
Ok thanks, noted that